src/components/Team/MemberCard.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { BrowserRouter as Router, Route } from "react-router-dom";
+import "../../utility/fonts.css";
+import styled from 'styled-components';
+import SubpageOuter from "../SubpageOuter/SubpageOuter.jsx";
+import devices from '../../utility/MediaQueries.js';
+import {StaticP, StaticH1} from "../../utility/ContentStyles.js";
+import './MemberCard.css';
+
+
+
+var cardStyle = {
+  width: '100%'
+};
+
+class MemberCard extends React.Component {
+
+    constructor(props){
+        super(props);
+        this.state = {
```

Comment:
I don't see `this.state.imagePath` accessed anywhere. I think this constructor can be removed.